.github/workflows/run_test.yml

```diff
@@ -115,6 +115,13 @@ jobs:
                   command: 'npm install'
                   working-directory: ${{ matrix.test.path }}
 
+            - name: Install experimental dependencies for stable api tests if running in experimental mode
+              if: matrix.azle_source == 'link' && ${{ inputs.run-experimental }}
```

Comment:
What happens if azle_source is npm though? Don't we need to install azle-experimental-deps then as well?